<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demalex SQL manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../sdk/icons.qrc">
    <normaloff>:/icons/cc_64.png</normaloff>:/icons/cc_64.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTreeView" name="tvDatabaseStructure">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuConnections">
    <property name="title">
     <string>Connections</string>
    </property>
    <addaction name="aAddDatabase"/>
    <addaction name="aRemoveDatabase"/>
    <addaction name="aEditDatabase"/>
   </widget>
   <addaction name="mnuConnections"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="aAddDatabase"/>
   <addaction name="aRemoveDatabase"/>
   <addaction name="aEditDatabase"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="aAddDatabase">
   <property name="icon">
    <iconset resource="../sdk/icons.qrc">
     <normaloff>:/icons/database_add.png</normaloff>:/icons/database_add.png</iconset>
   </property>
   <property name="text">
    <string>Register database</string>
   </property>
   <property name="toolTip">
    <string>Register database</string>
   </property>
  </action>
  <action name="aRemoveDatabase">
   <property name="icon">
    <iconset resource="../sdk/icons.qrc">
     <normaloff>:/icons/database_delete.png</normaloff>:/icons/database_delete.png</iconset>
   </property>
   <property name="text">
    <string>Unregister database</string>
   </property>
   <property name="toolTip">
    <string>Unregister database</string>
   </property>
  </action>
  <action name="aEditDatabase">
   <property name="icon">
    <iconset resource="../sdk/icons.qrc">
     <normaloff>:/icons/database_edit.png</normaloff>:/icons/database_edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit connection</string>
   </property>
   <property name="toolTip">
    <string>Edit connection preperties</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../sdk/icons.qrc"/>
  <include location="../sdk/icons.qrc"/>
  <include location="../sdk/icons.qrc"/>
  <include location="../sdk/icons.qrc"/>
  <include location="../sdk/icons.qrc"/>
  <include location="../sdk/icons.qrc"/>
 </resources>
 <connections/>
</ui>
